telemetry/definitions/commonDefinitions.json

```diff
@@ -52,6 +52,16 @@
             "type": "string",
             "description": "A generic version metadata"
         },
+        {
+            "name": "operation",
```

Comment:
operation sounds like something we might want consistency between Toolkits. If that is the case, lets make this a definition that has allowed values for Add/Edit/Delete, and give it a less general name.

Reply:
Would 'modificationType' be a better alternative here?